tutorial.md

```diff
@@ -0,0 +1,191 @@
+# Hello Cloud Demo Step-by-Step Tutorial
+
+This is a step-by-step tutorial on how to build and deploy this project to the cloud
+
+---
+
+# <a name="prereq"></a> Pre-Requisites
+
+- Docker
+- DockerHub Account. Click [here](https://hub.docker.com/) to create account
+- Java
+- Git
+- curl
+- Maven
+- helm. Install helm by following the instructions [here](https://helm.sh/docs/intro/install/)
+- Cloudant database
+- IBM Cloud Account. Click [here](https://cloud.ibm.com) to create account
+- IBM cloud CLI (IBM cloud command line interface)
+- kubectl (kubernetes command line interface)
+- Kubernetes Cluster
+
+---
+
+### If you already have the above pre-requisites, you can jump to [Step-by-Step](#stepbystep)
+
+---
+
+### 0. Make sure you created the accounts highlighted in [Pre-Requisites](#prereq), they are DockerHub and IBM Cloud Account, and install helm.
+
+### 1. Install IBM cloud CLI interface ibmcloud 
+```
+curl -fsSL https://clis.cloud.ibm.com/install/osx | sh
+```
+Note: This instructions assumes user has Mac OSX. To install IBM Cloud CLI in another platform please visit [platforms](https://cloud.ibm.com/docs/cli?topic=cli-install-ibmcloud-cli) and check your OS equivalent command line option to install ibmcloud CLI
```

Comment:
> Note: This instructions assumes user has Mac OSX. To install IBM Cloud CLI in another platform please visit [platforms](https://cloud.ibm.com/docs/cli?topic=cli-install-ibmcloud-cli) and check your OS equivalent command line option to install ibmcloud CLI

edit:
Note: This instruction assumes you have Mac OSX. Please visit [Install IBM Cloud CLI](https://cloud.ibm.com/docs/cli?topic=cli-install-ibmcloud-cli) and check your OS equivalent steps to install IBM Cloud CLI for your platform.

Let's move this note above the curl command so the reader is aware before they run the command